src/crypto/sha256_arm_shani.cpp

```diff
@@ -0,0 +1,898 @@
+// Copyright (c) 2018-2020 The Bitcoin Core developers
```

Comment:
was this moved from somewhere? Otherwise it should probably be updated

Reply:
Addresssed in the latest rebase